### ISSUE TITLE
Session are not saved with rediscluster

### ISSUE THREAD

**Issue by NoNalTrX** [phpredis/phpredis#1199]
I'd like to save my session in a redis cluster, but nothing is saved.

My actual config is: 

```
session.save_handler = rediscluster
session.save_path = "seed[]=node1:6379&seed[]=node2:6379&seed[]=node3:6379"
```

my test code is:

```
session_start();
print_r($_SESSION); 
echo "session = " . $_SESSION["test"]; 
$_SESSION["test"] = "I'm a test";
```

I got the error log:

`session_start(): Failed to read session data: rediscluster (path: seed[]=node1:6379&seed[]=node2:6379&seed[]=node3:6379)`

I'm on
- OS: Ubuntu Server 16
- Redis: 3.0.6
- PHP: 7.1.6
- phpredis: 3.1.2

I've checked my redis cluster by saving data in it using:

```
 $obj_cluster = new RedisCluster(NULL, Array("localhost:6379"), 1.5, 1.5, true);
 
  echo "test : " . $obj_cluster->get('test');
  $obj_cluster->set('test', "I'm a test");
```

It works perfectly.

**Comment by yatsukhnenko** [comment]
@NoNalTrX 
> Failed to read session data: rediscluster (path: seed[]=node1:6379&seed[]=node2:6379&seed[]=node3:6379)

node1, node2 and node3 are servers in your network or you just forgot to change addresses? :joy:

**Comment by NoNalTrX** [comment]
No I've just anonymised the domain name in the issue to avoid long string with the domain names.

**Comment by yatsukhnenko** [comment]
> $obj_cluster = new RedisCluster(NULL, Array("localhost:6379"), 1.5, 1.5, true);

Is this code works with that servers but not localhost?

**Comment by NoNalTrX** [comment]
Yes, it does. I tried with domain name, IP address and localhost as localhost is one of the node of the cluster. 
 All those solution works fine. In fact my entire app use Redis Cluster as backend cache. Just the sessions failing.

**Comment by yatsukhnenko** [comment]
@NoNalTrX, could you test fix from [issue.1199](https://github.com/phpredis/phpredis/tree/issue.1199) branch

**Comment by NoNalTrX** [comment]
the session seems to be working using this branch. Thanks.

**Comment by NoNalTrX** [comment]
do you have an idea of the time it will take to get this fix merged and packaged in the debian package?

**Comment by yatsukhnenko** [comment]
Honestly I don't know who, how and when creating debian packages... but I'm going to publish this fix as part of 3.1.3RC2 in a two or three weeks

**Comment by NoNalTrX** [comment]
Ok, thanks. And thanks for the fix. 👍
